pilot/pkg/networking/core/cluster_builder_test.go

```diff
@@ -1405,9 +1405,13 @@ func TestClusterDnsLookupFamily(t *testing.T) {
 				Attributes:   model.ServiceAttributes{Name: "svc", Namespace: "default"},
 			}
 			defaultCluster := cb.buildCluster(tt.clusterName, tt.discovery, endpoints, model.TrafficDirectionOutbound, servicePort, service, nil, "")
+			c := defaultCluster.build()
 
-			if defaultCluster.build().DnsLookupFamily != tt.expectedFamily {
-				t.Errorf("Unexpected DnsLookupFamily, got: %v, want: %v", defaultCluster.build().DnsLookupFamily, tt.expectedFamily)
+			if c.DnsLookupFamily != tt.expectedFamily {
+				t.Errorf("Unexpected DnsLookupFamily, got: %v, want: %v", c.DnsLookupFamily, tt.expectedFamily)
+			}
+			if c.DnsJitter.AsDuration() <= 0 {
```

Comment:
```suggestion
			if c.DnsJitter.AsDuration() <= 0 {
```

Setting to 0 to disable makes sense to me, but that means this won't work, correct?

Reply:
I know this is already merged, but I can follow it up with another PR. My thought process was that the default should be positive, but 0 is a valid value.